src/datascience-ui/common/cellFactory.ts

```diff
@@ -1,195 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-'use strict';
-
-import type * as nbformat from '@jupyterlab/nbformat';
-import { KernelFailure } from '../../platform/errors/errorUtils';
-// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
-const cloneDeep = require('lodash/cloneDeep');
-import '../../platform/common/extensions';
-import { IJupyterSettings } from '../../platform/common/types';
-import { CellMatcher } from '../../platform/datascience/cellMatcher';
-import { appendLineFeed, generateMarkdownFromCodeLines, removeLinesFromFrontAndBackNoConcat } from './index';
-
-export function uncommentMagicCommands(line: string): string {
```

Comment:
There was already a file called cellFactory.ts. And it turns out none of these are used by the UI so I moved them to other locations.